Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -349,6 +344,15 @@ public struct TargetSourcesBuilder {
             }
         }
     }
+    
+    private func diagnoseInvalidResource(in resources: [Resource]) {
+        resources.forEach { resource in
+            if let message = validTargetPath(at: resource.path) {
+                let warning = "Invalid Resource: \(message) '\(resource.path)'"
```

Comment:
I think that if we reverse this, it will be more readable: "Invalid resource \(resource): \(message)."